src/coreclr/vm/prestub.cpp

```diff
@@ -1569,6 +2228,12 @@ Stub * CreateInstantiatingILStub(MethodDesc* pTargetMD, void* pHiddenArg)
     // InstantiatingStub
     pCode->EmitLDC((TADDR)pHiddenArg);
 
+    // 2.3.2 Push the async continuation
```

Comment:
Is there a point in numbering the comments here? Delete the numbering on all comments instead?